docs/pages/components/toolbar.md

```diff
@@ -0,0 +1,220 @@
+---
+title: Toolbar
+id: toolbar
+keywords: toolbar
+sidebar: left-navigation-sidebar
+toc: false
+permalink: components/toolbar.html
+folder: components
+summary:
+---
+
+The toolbar enables the user to change the UI or trigger an action. For example, the toolbar allows the user to change views, manipulate data or objects, navigate to another page, perform generic actions, and so on.
+{: .docs-intro}
+
+## Toolbar types
+
+### Solid
+
+{% capture toolbar %}
+
+  <div class="fd-toolbar fd-toolbar--active fd-toolbar--solid">
+    <span>Title</span>
+    <button class="fd-button fd-button--compact">Button</button>
```

Comment:
These are transparent type buttons in the specs, I think we should match that design

Reply:
Right, fixed ;)